src/explanations/pancakeswap.yaml

```diff
@@ -0,0 +1,113 @@
+
+
+    ### Pottery
+    
+    
+    Pottery is a yield farming strategy on PancakeSwap that allows users to earn CAKE, the platform's native token, by depositing CAKE and participating in a lottery-style drawing. To participate in Pottery, users must deposit a minimum of 1 CAKE on the Pottery page during a specified period, which occurs on the Friday before the first Monday of each month. The deposited CAKE is directed to a lock-staking pool and locked for ten weeks. During this time, 80% of the total staking rewards generated from the deposited CAKE are directed to a prize pool for weekly drawings, and the remaining 20% are reserved for withdrawal after the ten weeks are complete. The chance of winning a prize in the weekly drawings is determined by the size of the user's deposit relative to the overall pool. Pottery uses Chainlink's implementation of VRF (Verifiable Random Function) to ensure the security and randomness of the drawing process. Pottery is a high-risk activity and can be volatile, so it is important for users to thoroughly research and understand the risks involved before participating.
+    
+    
+    ### The NFT Marketplace
+
+
+    Pancakeswap has its own NFT marketplace where users can buy and sell non-fungible tokens (NFTs). The NFTs on the platform are divided into different categories, such as art, collectibles, and gaming. The NFT marketplace has become very popular on Pancakeswap, with users buying and selling NFTs for a wide range of purposes.
+
+
+    ### The NFT Profile System
+
+
+    Pancakeswap also has an NFT profile system that allows users to showcase their NFT collection. The NFT profile system allows users to display their NFTs in a virtual gallery, and other users can view and interact with the NFTs. The NFT profile system has become very popular on Pancakeswap, with users using it to show off their NFT collections and discover new NFTs.
+
+
+    ### Initial Farm Offerings
+
+    Pancakeswap also has initial farm offerings (IFOs) that allow users to earn tokens by providing liquidity to a new project. IFO is a way for the platform to raise funds and create initial liquidity for the CAKE token. The IFOs on Pancakeswap are usually open for a limited time, and users who provide liquidity during the IFO period are rewarded with tokens. Pancakeswap uses a "dutch auction" format for its IFOs. This means that the price of CAKE decreases over time until the IFO ends, and users can purchase CAKE at the lowest price reached during the IFO. This allows users to purchase CAKE at a discount compared to the price it will be traded at once the IFO ends.
+
+- title: CAKE Token
+  shortTitle: CAKE
+  key: cake-token
+  details: |
+    CAKE is a cryptocurrency token that is used on the Pancakeswap decentralized exchange (DEX). It is a Binance Smart Chain (BSC) token and is used for a variety of purposes on the Pancakeswap platform, including staking, governance, and liquidity provision.
```

Comment:
Seems like there are multiple types of CAKE tokens also like vCAKE, iCAKE etc 

https://medium.com/pancakeswap/introducing-vcake-weighted-voting-power-based-on-fixed-term-cake-staking-a8ea6c385b81